### DISCUSSION TITLE
Restore contact-point, notification policy using export option

### DISCUSSION THREAD

**Discussion by rdwr-vijaykumarb** [grafana/grafana#64296] [Alerting]
Could you please add export feature (only for alerts export option is available ) for contact-point, notification policy which is not available in 9.4.2

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82800. Feel free to continue the discussion around this there. Thank you!
